### ISSUE TITLE
The error "The path is not a legal form" in Json connection in Designer Community edition

### ISSUE THREAD

**Issue by fediachov** [FastReports/FastReport#52]
![json-error](https://user-images.githubusercontent.com/43993341/49925604-7daae200-feca-11e8-876b-7f3a69e320a0.PNG)
An error is returned when you specify any file path in a connection editor. This bug exists only in FastReport Desginer Community Edition.
![json-error-version](https://user-images.githubusercontent.com/43993341/49925966-818b3400-fecb-11e8-8f13-1a45f955e00e.PNG)

**Comment by fediachov** [comment]
When using the Json from code:
![json-error2](https://user-images.githubusercontent.com/43993341/49930286-50affc80-fed5-11e8-83fa-2109b61a7be9.PNG)
Json data for testing:
[json-db.zip](https://github.com/FastReports/FastReport/files/2675395/json-db.zip)

**Comment by Detrav** [comment]
Hi, there is a problem because of the new load assembly system that is used in projects `<Project Sdk="Microsoft.NET.Sdk">`. 

Dependency files are no longer copied and there is no global cache; they are now in their own directory. Therefore, you need to add the property `<CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>` to work with plugins.

For example:
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp2.1</TargetFramework>
    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="FastReport.OpenSource" Version="2018.4.16" />
    <PackageReference Include="FastReport.OpenSource.Data.Json" Version="2018.4.16" />
    <PackageReference Include="Newtonsoft.Json" Version="12.0.1" />
  </ItemGroup>

</Project>
```

**Comment by Detrav** [comment]
Such an error is present in FastReport Desginer Community Edition because there are too many plugins connected to the designer and some of them use different versions of the same libraries.

    Getting Out of DLL Hell

We are solving this problem. As soon as there is news I will inform.

**Comment by Detrav** [comment]
Fixed in [2019.1.0](https://github.com/FastReports/FastReport/releases)

**Comment by fediachov** [comment]
Thank you!
